README.md

```diff
@@ -11,7 +11,6 @@
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Run](#run)
   - [Usage](#usage)
   - [Test](#tests)
 - [👥 Authors](#authors)
```

Comment:
- [ ] Your MIT license link redirects to an error 404 page. It is not a good practice to allow broken links in our README. Kindly fix this.
![Screen Shot 2023-03-16 at 11 28 30 PM](https://user-images.githubusercontent.com/62849872/225831173-a24fe66c-33ec-4c5c-a8d9-2026810d97d8.png)